src/main/java/Ui.java

```diff
@@ -0,0 +1,198 @@
+                System.out.println(LINE);
+
+            } catch (DukeException e) {
+                System.out.println(e.getMessage());
+                System.out.println(LINE);
+            }
+        }
+
+        sc.close();
+        end();
+    }
+
+    /**
+     * Greeting of users to signify the start of the program.
+     */
+    public void greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        String start = "Hello! I'm Duke, your personal assistant. \nWhat can I do for you?";
+        System.out.println(LINE);
+        System.out.println(logo);
+        System.out.println(start);
+        System.out.println(LINE);
+    }
+
+    /**
+     * Good bye to signify the end of the program.
+     */
+    public void end() {
+        String end = "Goodbye! Hope to see you again soon. :)";
+        System.out.println(LINE);
+        System.out.println(end);
+        System.out.println(LINE);
+    }
+
+    /**
+     * Prints out the current list of tasks.
+     */
+    public void list() {
+        if (list.isEmpty()) {
+            System.out.println("You have no tasks, go out and have fun! ~");
+        } else {
+            System.out.println("Here is your to-do list:\n");
+            for (int i = 1; i <= list.getTotalTasks(); i++) {
+                System.out.println(i + ". " + list.get(i - 1).toString());
+            }
+        }
+    }
+
+    /**
+     * Marks the task indicated as completed and prints out the task.
+     * that was completed.
+     * @param taskNumber Task's number in the list.
+     */
+    public void done(int taskNumber) {
+        list.getList().get(taskNumber - 1).completed();
+        System.out.println("You've completed this task:");
+        System.out.println((list.getList().get(taskNumber - 1)).toString());
+    }
+
+    /**
+     * Removes the task indicated from the list and prints out the task.
+     * @param taskNumber Task's number in the list.
+     */
+    public void delete(int taskNumber) {
+        String deleted = (list.getList().get(taskNumber - 1)).toString();
+        list.remove(taskNumber - 1);
+        System.out.println("Ok, this task has been kicked off your to-do list:");
+        System.out.println(deleted);
+        System.out.println(list.toString());
+    }
+
+    /**
+     * Creates a new todo task.
+     * @param task Task details.
+     */
+    public void todo(String task) {
+        Todo todo = new Todo(task, false);
+        list.add(todo);
+        System.out.println("I've added this task:\n");
+        System.out.println(todo.toString());
+        System.out.println(list.toString());
+    }
+
+    /**
+     * Creates a new event.
+     * @param task Event details.
+     * @param date Date of event.
+     */
+    public void event(String task, String date) {
```

Comment:
Method names should contain verbs, seems like some other methods have this issue too